conf/resources/data/GB000011/18AT320000TVWJYM69/response.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We don't need this response - 15 is not a valid response code, it is just an example from the docs. Only the ones in the confluence page are required.